docs/bar.js.html

```diff
@@ -917,6 +918,22 @@ <h1 class="page-title">Source: bar.js</h1>
         };
 
         /**
+         * Gets or Sets the padding of the chart
+         * Default is 0.1
+         * @param  {Number} _x Padding value to get/set
+         * @return { padding | module} Current padding or Chart module to chain calls
```

Comment:
This is wrong in a lot of our comments, it should be:
   @return { Number | module} Current....

Sorry for the confusion, we are fixing those as we see them!